plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include <fc/io/json.hpp>
 #include <fc/variant.hpp>
+#include <fc/crypto/hex.hpp>
```

Comment:
This include file not needed.

Reply:
agree !  leftover from last use of `from_hex( const fc::string& hex_str, char* out_data, size_t out_data_len )`  , thank you!